res/app/control-panes/screenshots/screenshots-controller.js

```diff
@@ -18,10 +20,25 @@ module.exports = function ScreenshotsCtrl($scope) {
     $scope.control.screenshot().then(function (result) {
       $scope.$apply(function () {
         $scope.screenshots.unshift(result)
+        if ($scope.screenshots.length > $scope.maxScreenshots){
+          $scope.screenshots.pop()
+        }
       })
     })
   }
 
+  $scope.takeScreenShotContinous = function(){
+    if ($scope.capturePromise) {
+      $interval.cancel($scope.capturePromise)
+      $scope.capturePromise = null
+      return
+    }
+
+    $scope.capturePromise = $interval(function(){
```

Comment:
You need to make sure that this gets cancelled when the scope gets destroyed. See https://docs.angularjs.org/api/ng/service/$interval.